### PR TITLE
Remove dead links

### PR DESCRIPTION
**Pull request by freearhey**
All links tested with `npm test` and VLC player

### CODE REVIEW

File: channels/es.m3u [hunk 1]
```diff
@@ -315,8 +315,6 @@ http://5.189.181.203:1935/gcm4/gcm4/chunklist.m3u8
 http://ventdelnord.tv:8080/girona/directe.m3u8?fluxustv.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://media.tvalacarta.info/canales/256x256/tvmelilla.png" group-title="",Televisión Melilla
 https://melillatv-hls.flumotion.com/melillatv/melillatv_all/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/OKxWzyO.png" group-title="",Tenerife 2030
-http://livestream.iter.es:9876/tdt/tdt.m3u8
```
Comment (boomski) [inline channels/es.m3u]: Link works (Tested in VLC)

File: channels/rs.m3u [hunk 2]
```diff
@@ -3,8 +3,6 @@
 https://sportbar.biz/lba/123/g6/arena1rs.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/KHRWK6Y.png" group-title="Sport",ARENA SPORT 3
 https://sportbar.biz/lba/123/g1/sportarena3rs.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/oKLqSYI.png" group-title="Sport",ARENA SPORT 4
-https://sportbar.biz/lba/123/g3/arenaserbia4.m3u8
```
Comment (boomski) [inline channels/rs.m3u]: Link works (Tested in VLC)

File: channels/xk.m3u [hunk 3]
```diff
@@ -31,11 +31,7 @@ http://stream2.rtkit.com:1935/rtk4stream/rtk4.stream/chunklist_w599618358.m3u8
...
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/KJLm3gi.jpg" group-title="",RTV ZiK
 http://uk4.streamingpulse.com:1935/RTVZiK/RTVZiK/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/0qVIOSH.png" group-title="",Syri Vision
-rtmp://37.26.69.49:80/live/syri
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/gfdZq6D.jpg" group-title="",TV Dielli
 http://51.15.180.159:8081/play/hls/dielli/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/C1uuCgo.png" group-title="",TV Opoja
 http://163.172.211.229:8000/live/sula/sula123/77.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/C1uuCgo.png" group-title="",TV Opoja
-http://ip.opoja.tv:1935/tvopoja/tvopoja/playlist.m3u8
```
Comment (boomski) [inline channels/xk.m3u]: Link works (Tested in VLC)

File: channels/mx.m3u [hunk 4]
```diff
@@ -5,8 +5,6 @@ http://live.gvstream.net:1935/e-tv-video/play.stream/playlist.m3u8
 http://aztkhlslive3-i.akamaihd.net/hls/live/777407/adn40mx/master01.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/9Xn7ZEZ.jpg" group-title="",Alcarria TV
 http://217.182.77.27/live/alcarriatv-livestream.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/rbDb5mb.png" group-title="",Aprende TV
-https://srvstream01.televisioneducativa.gob.mx/hls/aprendetv.m3u8
```
Comment (boomski) [inline channels/mx.m3u]: Link works (Tested in VLC)

File: channels/mx.m3u [hunk 5]
```diff
@@ -43,8 +37,6 @@ http://live.canaloncelive.tv:1935/livepkgr2/smil:internacional.smil/playlist.m3u
 https://180ce25c70.site.internapcdn.net/ryta_live/chunklist_b481072.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/NAgRCyY.png" group-title="",Canal Once
 https://live2.canaloncelive.tv/livepkgr2/smil:internacional.smil/chunklist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/857zxbL.png" group-title="",Capacita TV
-https://srvstream01.televisioneducativa.gob.mx/hls/capacitatv.m3u8
```
Comment (boomski) [inline channels/mx.m3u]: Link works (Tested in VLC)

File: channels/mx.m3u [hunk 6]
```diff
@@ -59,8 +51,6 @@ http://d3kr0d4mfjxpbv.cloudfront.net/p/SPAS_1920x1080_4000k.m3u8
 https://d2416wdkjdz9sf.cloudfront.net/forotv/forotvm.m3u8?fluxustv.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/hMtKE60.png" group-title="",Genesis TV
 http://201.144.184.98:1935/genesis/smil:television.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/CA0LESg.jpg" group-title="",Ingenio TV
-https://srvstream01.televisioneducativa.gob.mx/hls/ingeniotv.m3u8
```
Comment (boomski) [inline channels/mx.m3u]: Link works (Tested in VLC)

File: channels/mx.m3u [hunk 7]
```diff
@@ -115,8 +105,6 @@ http://wms.tecnoxia.com:1935/rytqrolive/rytqrolive/master.m3u8?GrupoTopIptv
 http://streamingcws20.com:1935/sqcs/videosqcs/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/FHiX8SL.png" group-title="",Tele Yucatan
 http://streamingcws20.com:1935/trecevision/videotrecevision/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/whNDApb.png" group-title="",Telebachillerato
-https://srvstream01.televisioneducativa.gob.mx/hls/telebachillerato.m3u8
```
Comment (boomski) [inline channels/mx.m3u]: Link works (Tested in VLC)

File: channels/mx.m3u [hunk 8]
```diff
@@ -129,10 +117,6 @@ http://wms.tecnoxia.com:1935/8006/8006/playlist.m3u8
 http://s5.mexside.net:1935/telemax/telemax/.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/0gL3FIN.png" group-title="",Teleritmo
 http://mdstrm.com/live-stream-playlist_400/57b4dc126338448314449d0c.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/qf0jh1I.png" group-title="",Telesecundaria
-https://srvstream01.televisioneducativa.gob.mx/hls/telesecundaria.m3u8
```
Comment (boomski) [inline channels/mx.m3u]: Link works (Tested in VLC)

**Review by boomski — CHANGES_REQUESTED** [review]
Some channels are working tested on VLC

File: channels/es.m3u (hunk 1 above)
Comment (freearhey) [reply channels/es.m3u]: Strange, for me the channel is still not working. Here for example what gives out VLC player:
Screenshot 2019-10-12 at 20 04 55

File: channels/xk.m3u (hunk 3 above)
Comment (freearhey) [reply channels/xk.m3u]: Still not working. VLC player produces:
Screenshot 2019-10-12 at 20 12 06

File: channels/mx.m3u (hunk 4 above)
Comment (freearhey) [reply channels/mx.m3u]: Still not working. VLC output:
Screenshot 2019-10-12 at 20 14 22

File: channels/mx.m3u (hunk 5 above)
Comment (freearhey) [reply channels/mx.m3u]: Still not working. VLC output:
Screenshot 2019-10-12 at 20 16 08

File: channels/mx.m3u (hunk 6 above)
Comment (freearhey) [reply channels/mx.m3u]: Still not working. VLC output:
Screenshot 2019-10-12 at 20 17 20

File: channels/mx.m3u (hunk 7 above)
Comment (freearhey) [reply channels/mx.m3u]: Still not working. VLC output:
Screenshot 2019-10-12 at 20 18 06

File: channels/mx.m3u (hunk 8 above)
Comment (freearhey) [reply channels/mx.m3u]: Still not working. VLC output:
Screenshot 2019-10-12 at 20 19 14

**Comment by freearhey** [comment]
Returned "ARENA SPORT 4" to `rs.m3u` playlist. The remaining channels still do not work for me.

File: channels/xk.m3u (hunk 3 above)
Comment (boomski) [reply channels/xk.m3u]: For me this links plays in VLC  and also on streamtest.in
Screenshot_2019-10-12 Test Live Streaming Streamtest in

File: channels/mx.m3u (hunk 4 above)
Comment (boomski) [reply channels/mx.m3u]: This stream works for me in VLC 
Screenshot_2019-10-12 Test Live Streaming Streamtest in(1)

File: channels/mx.m3u (hunk 5 above)
Comment (boomski) [reply channels/mx.m3u]: Still works for me

File: channels/mx.m3u (hunk 6 above)
Comment (boomski) [reply channels/mx.m3u]: Still works for me

File: channels/mx.m3u (hunk 7 above)
Comment (boomski) [reply channels/mx.m3u]: Still works for me

File: channels/mx.m3u (hunk 8 above)
Comment (boomski) [reply channels/mx.m3u]: Still works for me

File: channels/es.m3u (hunk 1 above)
Comment (boomski) [reply channels/es.m3u]: afbeelding
- When i test this one now it is not playing in VLC , this morning it did.
- Right Now i did also test on streamtest.in ( see image) there it works , and also in kodi it plays. 
It is not a dead link , it is a VLC problem but i encounter more and more links with this issue.

Comment (boomski) [reply channels/es.m3u]: You could try this one : http://livestream.iter.es:9876/tdt/tdt_720.m3u8
This one i pulled directly from the website -> http://www.canaltenerifetv.com/channels/tenerife-tv
And try if you can play it in your browser

File: channels/xk.m3u (hunk 3 above)
Comment (boomski) [reply channels/xk.m3u]: https://54627d4fc5996.streamlock.net/tvopoja/tvopoja/chunklist.m3u8 - Try if this one works

File: channels/mx.m3u (hunk 4 above)
Comment (boomski) [reply channels/mx.m3u]: All the links starting with "https://srvstream01.televisioneducativa.gob.mx/hls"
are from this website 
https://www.televisioneducativa.gob.mx/#multimedia
try if you can play them there.

Comment (freearhey) [reply channels/mx.m3u]: I can’t access this site at all
Screenshot 2019-10-13 at 04 38 35

File: channels/xk.m3u (hunk 3 above)
Comment (freearhey) [reply channels/xk.m3u]: @boomski this link is working fine

File: channels/es.m3u (hunk 1 above)
Comment (freearhey) [reply channels/es.m3u]: http://livestream.iter.es:9876/tdt/tdt_720.m3u8 - doesn't work either
And the site also does not open:
Screenshot 2019-10-13 at 04 43 39

**Comment by freearhey** [comment]
In order not to waste time, I will return all the links that work for you for now. If something changes then we’ll delete it.

**Comment by boomski** [comment]
Ok so the spanish and mexican channels probably will have geoblock in som territory , if you even can not open the website.

**Review by boomski — APPROVED** [review]

**Pull request merged**